environments/icds-cas/inventory/couchdb2.csv

```diff
@@ -1,10 +1,11 @@
-hostname,host_address,group 1,group 2,group 3,var: hostname
-couch00,100.71.184.191,couchdb2,couchdb2_proxy,,MUMGCCWCDPRDCDV00
-couch01,100.71.184.192,couchdb2,,,MUMGCCWCDPRDCDV01
-couch02,100.71.184.193,couchdb2,,,MUMGCCWCDPRDCDV02
-couch03,100.71.184.194,couchdb2,,,MUMGCCWCDPRDCDV03
-couch04,100.71.184.195,couchdb2,,,MUMGCCWCDPRDCDV04
-couch05,100.71.184.196,couchdb2,,,MUMGCCWCDPRDCDV05
-couch06,100.71.184.197,couchdb2,,,MUMGCCWCDPRDCDV06
-couch07,100.71.184.198,couchdb2,,,MUMGCCWCDPRDCDV07
-couch08,100.71.184.199,couchdb2,,,MUMGCCWCDPRDCDV08
+hostname,host_address,group 1,group 2,group 3,var: keepalived_role,var: keepalived_shared_ip,var: keepalived_shared_iface,var: hostname
+couch00,100.71.184.191,couchdb2,couchdb2_proxy,,MASTER,100.71.184.124,ens160,MUMGCCWCDPRDCDV00
+couch09,100.71.184.200,couchdb2,couchdb2_proxy,,BACKUP,100.71.184.124,ens160,MUMGCCWCDPRDCDV09
```

Comment:
FYI adding this to the `couchdb2` group will add it to the couch cluster. I think since it's not currently part of the cluster we might as well leave it out for now.